From 8047952e5d4a5aad9d5a3a2aca05815eabc6d059 Mon Sep 17 00:00:00 2001
From: Nabi <artsanaat@gmail.com>
Date: Mon, 19 Jun 2023 11:56:15 +0330
Subject: [PATCH] Fixed various PHP notices in other plugins on edit feed 
 settings page caused by incompatible invocation of add_meta_boxes hook in 
 webappick-product-feed-for-woocommerce-pro.

---
 libs/webappick-product-feed-for-woocommerce/includes/helper.php | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libs/webappick-product-feed-for-woocommerce/includes/helper.php b/libs/webappick-product-feed-for-woocommerce/includes/helper.php
index d4e6977d..668d4611 100755
--- a/libs/webappick-product-feed-for-woocommerce/includes/helper.php
+++ b/libs/webappick-product-feed-for-woocommerce/includes/helper.php
@@ -914,7 +914,6 @@ if ( ! function_exists( 'woo_feed_register_and_do_woo_feed_meta_boxes' ) ) {
 		 * This action is documented in wp-admin/includes/meta-boxes.php
 		 * using screen id instead of post type
 		 */
-		do_action( "add_meta_boxes_{$screen->id}", $feedRules );
 		do_action( 'do_meta_boxes', $screen->id, 'normal', $feedRules );
 		do_action( 'do_meta_boxes', $screen->id, 'advanced', $feedRules );
 		do_action( 'do_meta_boxes', $screen->id, 'side', $feedRules );
-- 
2.15.0

